src/tools/perf/api/libperf.h

```diff
@@ -98,6 +98,29 @@ enum {
 };
 
 
+#define FOREACH_UCP_PERF_DAEMON_AM_ID \
+    /* Host to daemon communication */  \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_INIT,     0) \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_REQ,      1) \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_SEND_ACK, 2) \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_RECV_ACK, 3) \
```

Comment:
why request is not separated to send/recv but ack is separated to send/recv?

Reply:
yes, i think we need better names, something that looks like (name) and (name)_ACK

Reply:
Following our conversation, I updated all the ID names and improved the documentation, please check